packages/auth/src/core/util/handler.ts

```diff
@@ -107,7 +114,18 @@ export function _getRedirectUrl(
       delete paramsDict[key];
     }
   }
-  return `${getHandlerBase(auth)}?${querystring(paramsDict).slice(1)}`;
+
+  // Sets the App Check token to pass to the widget
+  const appCheckToken = await auth._getAppCheckToken();
+  const appCheckTokenFragment = appCheckToken
+    ? `${FIREBASE_APP_CHECK_FRAGMENT_ID}=${encodeURIComponent(appCheckToken)}`
+    : '';
+  const fragment = appCheckTokenFragment ? `#${appCheckTokenFragment}` : '';
```

Comment:
Since there's only ever one thing in the fragment for now, I'd suggest just putting the `#` directly on line 121 and getting rid of the `fragment` variable entirely.

Reply:
Sounds good, updated